formula-coroutines/src/main/java/com/instacart/formula/coroutines/FlowRuntimeExtensions.kt

```diff
@@ -0,0 +1,25 @@
+package com.instacart.formula.coroutines
+
+import com.instacart.formula.IFormula
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOf
+
+@ExperimentalCoroutinesApi
+fun <Output : Any> IFormula<Unit, Output>.toFlow(): Flow<Output> {
```

Comment:
If these are experimental I'd probably create your own experimental annotation instead of reusing `ExperimentalCoroutinesApi`. `ExperimentalFormulaApi` would follow the same convention.

Reply:
I don't think they are experimental (just implementation details) so I think the (more) correct approach would be to opt in and hide it from the consumer.

Reply:
@mattkranzler5 Hmm, since Formula depends on experimental (non-stable) APIs of coroutines, there might be an issue where a consumer updates coroutines to a version where these non-stable APIs have changed. This could cause a compilation issue or maybe even a runtime issue in Formula. If we opt-in, we are hiding this information from the consumer.

@colinrtwhite I don't have a good understanding yet of how these experimental annotations should be used. The public API of the `toFlow()` is not really experimental, it just relies on experimental APIs within its implementation. Is the best practice to create a custom annotation in those cases?

Reply:
If it's only using experimental coroutines APIs then I'd vote to use `@OptIn(ExperimentalCoroutinesApi::class)` as well. In practice, the experimental coroutines APIs go through a deprecation cycle before being removed so it's relatively safe to hide their usage (using opt in) as an implementation detail.

Reply:
Cool, will change it to `@OptIn`.